internal/pkg/agent/control/v2/server/server.go

```diff
@@ -77,6 +79,11 @@ func (s *Server) Start() error {
 	}
 	cproto.RegisterElasticAgentControlServer(s.server, s)
 
+	if startV1Wrapper {
```

Comment:
Why do we need to make this conditional?

Also can you add a comment or two explaining why we need the V1 wrapper so we can remember it in 6 months?